base/images/8.5-fpm/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# Debian Trixie provides a package of the MySQL client that prevent the connexion to succeed for SSL reasons
+# We stick to Bookworm until we find the solution
+FROM php:8.5-fpm-bookworm
```

Comment:
<img width="1485" height="950" alt="image" src="https://github.com/user-attachments/assets/fc8398e8-bc62-4130-8fff-a6875cfe6e53" />

<img width="1485" height="950" alt="image" src="https://github.com/user-attachments/assets/1257f138-3a94-4450-97dd-be434a3a3e00" />

The bug we recently encountered will be back with PHP 8.5 if we can't use Debian Bookworm :cry:

Reply:
It's time but regarding the debian distribution issue, is it available now? (to avoid reproducing our mysql bug)